<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUI</class>
 <widget class="QMainWindow" name="GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>802</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>775</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Live OPS Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="3,4">
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item row="2" column="0">
         <widget class="QwtTextLabel" name="_tempSliderLbl">
          <property name="plainText">
           <string>Temperature</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QwtSlider" name="_tempSlider">
          <property name="upperBound">
           <double>5.000000000000000</double>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QwtTextLabel" name="_gammaSliderLbl">
          <property name="plainText">
           <string>FvK Number</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QwtSlider" name="_gammaSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QwtSlider" name="_pressureSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="lowerBound">
           <double>-100.000000000000000</double>
          </property>
          <property name="totalSteps">
           <UInt>500</UInt>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QwtTextLabel" name="Pressure">
          <property name="plainText">
           <string>Pressure</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Turn on pressure</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="0">
         <widget class="QLabel" name="_fvkLbl">
          <property name="text">
           <string>FvK:</string>
          </property>
          <property name="buddy">
           <cstring>_fvkValLbl</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="_fvkValLbl">
          <property name="text">
           <string>0.01</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="_tempLbl_2">
          <property name="text">
           <string>Temperature:</string>
          </property>
          <property name="buddy">
           <cstring>_tempValLbl</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="_tempValLbl">
          <property name="text">
           <string>0.10</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="_pressureLbl">
          <property name="text">
           <string>Pressure</string>
          </property>
          <property name="buddy">
           <cstring>_pressureValLbl</cstring>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="_pressureValLbl">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="_timeStepLbl">
          <property name="text">
           <string>Time Step:</string>
          </property>
          <property name="buddy">
           <cstring>_timeStepValLbl</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="_timeStepValLbl">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QVTKOpenGLWidget" name="_qVTK">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="_initBtn">
          <property name="text">
           <string>Initialize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_startStopBtn">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTabWidget" name="_topPlotTab">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="_angleDeficitTab">
         <attribute name="title">
          <string>Angle Deficit</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QwtPlot" name="_angleDeficitPlot"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="_volumeTab">
         <attribute name="title">
          <string>Volume</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QwtPlot" name="_volumePlot"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="_bottomPlotTab">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="_energyTab">
         <attribute name="title">
          <string>Energy</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QwtPlot" name="_energyPlot"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_tempSlider</sender>
   <signal>valueChanged(double)</signal>
   <receiver>_tempValLbl</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>613</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_gammaSlider</sender>
   <signal>valueChanged(double)</signal>
   <receiver>_fvkValLbl</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>584</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_pressureSlider</sender>
   <signal>valueChanged(double)</signal>
   <receiver>_pressureValLbl</receiver>
   <slot>setNum(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>515</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>649</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
